Makefile

```diff
@@ -1,3 +1,4 @@
+SHELL=bash
```

Comment:
this might require some downstream packaging changes, I think even sandboxed linux builders aren't guaranteed to have bash without a build time dep. Not a big deal but just curious why?

Reply:
largely because I was debugging things.  I intend to swap that back out to sh which you are correct as being the only shell I can assert the presence of.

Initially I was using bash to do some stuff I've since backed out where expansions were more convenient, but the further I got into it the more I realized the tests were in need of discussion and agreement more than code.